### ISSUE TITLE
Changing exposure time during fly scan

### ISSUE THREAD

**Issue by decarlof** [tomography/tomoscan#135]
We are in the process of adding to tomoStream the ability to change angle step and exposure time during the fly-scan. For this we are adding call backs in tomoscan_stream_pso.py but there is a conflict with an existing call back in the base tomoscan.py class.

[pv_callback()](https://github.com/decarlof/tomoscan/blob/85de393eade3914cce6b8d5e8ea05ddbc2bdd393/tomoscan/tomoscan.py#L265) as written allows for a change of Exposure Time during a scan. 

I think the intention of this callback is to update the exposure time to area detector before starting a scan. If this is the case I would change the current [set_exposure_time() ](https://github.com/decarlof/tomoscan/blob/85de393eade3914cce6b8d5e8ea05ddbc2bdd393/tomoscan/tomoscan.py#L519) from:

```
        if exposure_time is None:
            exposure_time = self.epics_pvs['ExposureTime'].value
        self.epics_pvs['CamAcquireTime'].put(exposure_time, wait=True, timeout = 10.0)

```

to

```
        if not self.scan_is_running:
            if exposure_time is None:
                exposure_time = self.epics_pvs['ExposureTime'].value
            self.epics_pvs['CamAcquireTime'].put(exposure_time, wait=True, timeout = 10.0)
```

**Comment by MarkRivers** [comment]
That seems reasonable.  The intention of the current callback is so the user can set the exposure time in the TomoScan OPI screen and have it set the value in areaDetector.  It is intended to be used during setup, not when a scan is actually running.

**Comment by decarlof** [comment]
implemented in #136
